R-proj/src/copula.cpp

```diff
@@ -7,6 +7,7 @@
 
 //Contributed and/or modified by Apostolos Chalkis, as part of Google Summer of Code 2018 program.
 
+
```

Comment:
many files have this empty line insertion, could you undo these changes?

Reply:
done in `root/R-proj` folder